src/routes/countries.route.jsx

```diff
@@ -0,0 +1,37 @@
+import { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { getCoutries } from '../redux/countries';
+import CountryCard from '@/components/country-card.component';
+import './styles/countries.styles.scss';
+
+function Countries() {
+  const { status, countries } = useSelector((state) => state.countries);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    if (status === 'update') {
+      dispatch(getCoutries());
+    }
+  }, [status]);
+
+  return (
+    <div className="countries-container">
+      <div className="header">
+        <h1>AIR POLLUTION</h1>
+        <span>the real-time Air Quality</span>
+      </div>
+      <div className="bar">
+        <span>START BY SELECT A COUNTRY TO SEE MORE DETAIL</span>
+      </div>
+      <div className="countries">
+        {
+          countries.map((country) => (
+            <CountryCard key={country.id} country={country} />
+          ))
+        }
+      </div>
+    </div>
+  );
+}
+
+export default Countries;
```

Comment:
- According to the project requirements, your app should `Use a Filter stateless component to filter data retrieved from the API.` Please add a feature that enables users to filter the countries by a parameter, either by name or any parameter of your choice. Please fix this 🙏

Reply:
Well resolved 💯  awesome stuff!!! 🥳🎆 

![image](https://user-images.githubusercontent.com/38088818/226092209-6db9c705-e227-4ce2-8cb3-5b9b88ed4542.png)


[OPTIONAL] It would be nice to make the search `case insensitive`. You can use the JavaScript's embedded method [toLowerCase()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toLowerCase) on the search input to make sure that even if a user types in capital letters they are still able to get the same results.